core/pom.xml

```diff
@@ -111,6 +107,12 @@
       <version>20110809</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>commons-httpclient</groupId>
+      <artifactId>commons-httpclient</artifactId>
+      <version>3.1</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Directly imported by a test in this module, so declared explicitly as a test dependency.